frontend/helpers/path-helpers.ts

```diff
@@ -46,3 +47,7 @@ export function productListPath(
       }
    }
 }
+
+export function ifixitOriginFromHost(host: string): string {
+   return host.match(/(?:\w*\.)*cominor\.com/) ? '' : IFIXIT_ORIGIN;
```

Comment:
I'm not sure what's being attempted here with the first part of this regex:

`(?:\w*\.)*`  The star there indicates this part is effectively optional and since we don't capture it, this part of the pattern seems unused.

I might test a suffix: `/\.(cominor\.com|ubreakit\.com)$/` instead because we know subdomains can be several levels deep.

Reply:
Yea the first part does seem kind of useless. I'll use your regex